### DISCUSSION TITLE
LLM + Raspberry pi and Raspberry pi pico

### DISCUSSION THREAD

**Discussion by shreesha345** [microsoft/PromptCraft-Robotics#19] [LLM + Aerial robots]
I wonder if there is a way where I can run a environment in raspberry pi pico or raspberry pi 4 so that ChatGPT can execute the code there as when I give instructions to it.

**Comment by Xj1567589354** [comment]
As long as you can generate code supported by the Raspberry Pi Pico and save it in the main function on this Raspberry Pi Pico, won't it be automatically allowed?
